src/stores/descriptions.raw.txt

```diff
@@ -89,11 +77,11 @@ SignalTooltip: Confirmed influenza hospital admissions per 100,000 people
 Description: This data shows the number of hospital admissions with lab-confirmed influenza each day. We source this data from the Report on Patient Impact and Hospital Capacity published by the US Department of Health & Human Services (HHS). 
 ---
 Name: COVID Deaths
-Id: jhu-csse
-Signal: deaths_7dav_incidence_prop
+Id: nchs-mortality
+Signal: deaths_covid_incidence_prop
 
 
-SignalTooltip: Newly reported COVID-19 deaths per 100,000 people, based on data from Johns Hopkins University
+SignalTooltip: Newly reported COVID-19 deaths per 100,000 people, based on NCHS mortality data.
 
 
-Description: This data shows the number of COVID-19 deaths newly reported each day. The signal is based on COVID-19 death counts compiled and made public by [a team at Johns Hopkins University](https://systems.jhu.edu/research/public-health/ncov/).
\ No newline at end of file
+Description: This data source of national provisional death counts is based on death certificate data received and coded by the National Center for Health Statistics (NCHS). This data is different from the death data available from USAFacts and JHU CSSE. Deaths are reported by the date they occur, not the date they are reported by local health departments, and data is frequently reissued as additional death certificates from recent weeks are received and tabulated.
```

Comment:
I know you copied this from the paragraph near the top of https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/nchs-mortality.html, but lets keep it signal-specific and as succinct as before:
```suggestion
Description: This data shows the number of COVID-19 deaths newly reported each week.  The signal is based on COVID-19 death counts compiled and made public by [the National Center for Health Statistics](https://www.cdc.gov/nchs/nvss/vsrr/COVID19/index.htm).
```
@carlynvandyke Any comments on the wording?